go/tasks/plugins/webapi/agent/plugin.go

```diff
@@ -162,6 +162,8 @@ func (p Plugin) Status(ctx context.Context, taskCtx webapi.StatusContext) (phase
 	taskInfo := &core.TaskInfo{}
 
 	switch resource.State {
+	case admin.State_PENDING:
+		return core.PhaseInfoQueuedWithTaskInfo(core.DefaultPhaseVersion, "Job is PENDING", taskInfo), nil
```

Comment:
I don't like the message. This message does not convey any information to the user. Can the agent send this message

Reply:
@kumare3 
https://github.com/flyteorg/flyteplugins/blob/503d1be56eeca69b29606082905d8c3f3dbb86d7/go/tasks/pluginmachinery/core/phase.go#L220-L226
Is this solution suitable?